metabase/metabase
# Show server timezone info in `now` custom expression helper text

Besides updating help text for the now command, here we move the `suggest` and `helper-text-strings` implementation files, along with their test files, out of the `metabase-lib` dir and into the `ExpressionEditorTextfield` component dir. This move is in line with keeping `metabase-lib` about query-related logic. Also, we avoid importing `metabase/lib` material into the `metabase-lib` namespace.

### How to Test

1. `+ New`
2. `Question`
3. Important: choose a Postgres database
4. Choose a table
5. Click on `Custom Column` icon
6. Type `now(` in the input

You should see the help text looking like below, that is, giving info on server time and timezone.

<img width="496" alt="image" src="https://user-images.githubusercontent.com/380816/205081071-d1f9445b-e516-4736-b8b6-50a8ad984c5f.png">
